plugins/by-name/efmls-configs/default.nix

```diff
@@ -21,8 +22,18 @@ lib.nixvim.plugins.mkNeovimPlugin {
       packageName = "efm-langserver";
     })
 
+    (
+      let
+        basePluginPath = [
+          "plugins"
+          "efmls-configs"
+        ];
+      in
+      lib.mkRenamedOptionModule (basePluginPath ++ [ "setup" ]) (basePluginPath ++ [ "languages" ])
```

Comment:
We also need to handle this rename in tests/test-sources/plugins/by-name/efmls-configs/default.nix (see CI failure)